### PR TITLE
Fix node_modules route

### PR DESCRIPTION
**Pull request by cmangeat**

### CODE REVIEW

File: admin/c2cgeoportal_admin/routes.py [hunk 1]
```diff
@@ -3,7 +3,10 @@
 
 
 def includeme(config):
-    config.add_static_view('node_modules', 'c2cgeoportal_admin:node_modules')
+    config.add_static_view('node_modules_for_insider', 'c2cgeoportal_admin:node_modules')
+    config.add_static_view(
+        'node_modules_for_outsider',
+        '{}:node_modules'.format(config.root_package.__name__))
```
Comment (arnaud-morvan) [inline admin/c2cgeoportal_admin/routes.py]: Are you sure it make sense to add two static views with same name but different path.

**Comment by arnaud-morvan** [comment]
I do not understand what you've done here, is it to the admin module to register routes for outsiders ?

**Comment by cmangeat** [comment]
https://github.com/camptocamp/c2cgeoform/blob/master/c2cgeoform/__init__.py#L57 -> the outsider
https://github.com/camptocamp/c2cgeoportal/blob/master/admin/c2cgeoportal_admin/templates/layertree.jinja2#L4 -> one insider ?

**Comment by arnaud-morvan** [comment]
ok, thanks

**Pull request merged**